code/unclassified/src/apriori/apriori.py

```diff
@@ -0,0 +1,131 @@
+# Part of Cosmos by OpenGenus Foundation
+# Find frequently occuring subsets of items in a set of transactions
+# Contributed by: Aditya Lohana (dundermiflin)
+
+
+import csv
+from itertools import combinations, chain
+
+
+min_support = 0.1
+
+
+def csv_to_list(filename):
+	"""
+		Function to convert the transaction-dataset into a list of lists
+	"""
+	with open(filename, 'r') as f:
+		reader = csv.reader(f)
+		data = [row for row in reader]
```

Comment:
Instead of a list comprehension, just use the `list` function.

Reply:
Done, did not notice the redundancy.